surfer/tests/test_viz.py

```diff
@@ -37,6 +37,7 @@ def test_offscreen():
     """
     mlab.options.backend = 'auto'
     brain = Brain(*std_args, offscreen=True)
+    _ = brain.screenshot()
     shot = brain.screenshot()
     assert_array_equal(shot.shape, (800, 800, 3))
```

Comment:
I also get the same issue in the tests, and can fix it with taking an additional screenshot. Before adding the new line I got
```
AssertionError: 
Arrays are not equal

(mismatch 33.3333333333%)
 x: array([765, 800,   3])
 y: array([800, 800,   3])
```

Is that specific to my setup? I am using Mayavi from Anaconda (where Mayavi is still at 4.4.0 and forces Anaconda to 2.3) on OS X 10.11...